src/main/java/seedu/address/logic/parser/CalendarParser.java

```diff
@@ -44,20 +39,23 @@ public Command parseCommand(String userInput) throws ParseException {
         System.out.println("arguments " + arguments);
         switch (commandWord) {
 
-        case ListCommand.COMMAND_WORD:
-            return new ListCommand();
+            case ListCommand.COMMAND_WORD:
+                return new ListCommand();
+
+            case AddDeadlineCommand.COMMAND_WORD:
+                return new AddDeadlineCommandParser().parse(arguments);
 
-        case AddDeadlineCommand.COMMAND_WORD:
```

Comment:
The `case` statement needs to be on the same level of indentation as the `switch` statement. Need to configure under IntelliJ Settings -> Editor -> Code Style -> Java -> Wrapping and Braces. Edit on the right panel, if I recall correctly!

Reply:
Resolved in abf2cb4.